src/components/reviews/addReviewForm.jsx

```diff
@@ -57,6 +59,55 @@ class AddReviewForm extends React.Component {
     });
   }
 
+  readFileAsync(file) {
```

Comment:
We walked through this as a team. This method of accessing image files is really interesting.